### ISSUE TITLE
Bump thecodingmachine/safe to the latest version.

### ISSUE THREAD

**Issue by zemzale** [phpactor/indexer#21]
I been using the VIM plugin for phpactor. Today had some places where I wanted to use the `thecodingmachine/safe` package and had errors about redeclared versions of functions. 

Downgrading to the current version in this file worked just fine. but since v1.1.2 there have been some bug fixes.

**Comment by dantleech** [comment]
yes this is a problem (not just with Safe). Perhaps another solution is to remove the dependency on this library.

solutions are:

- remove the `Safe` library (not scalable as it  doesn't affect only this library)
- disable composer (` "composer.enable": false`): but this means some Phpactor features won't work (e.g. class generation etc)
- write a library which can resolve composer paths without including the composer autoloader ...
